bau/examples/bau-ts-test/src/main.ts

```diff
@@ -469,7 +469,7 @@ const TestDerived = () => {
     input({
       placeholder: "Enter username",
       value: inputState,
-      oninput: ({ target }: { target: HTMLInputElement }) =>
+      oninput: ({ target }) => target instanceof HTMLInputElement &&
```

Comment:
Why adding a runtime check ?

Reply:
Because it is safer and less verbose than a type assertion.

Reply:
Well since this is TypeScript I guess a type assertion is not less verbose